src/constants/index.ts

```diff
@@ -0,0 +1,9 @@
+export const DEFAULT_DOMAIN = 'meet.jit.si';
+
+export const DEFAULT_OPTIONS = {
+    roomName: 'DefaultRoomNameExample',
```

Comment:
Don't set a default room name please, people testing this may end up meeting strangers :-)